### ISSUE TITLE
README instructions result in an error

### ISSUE THREAD

**Issue by hsparra** [ivandorte/eo-notebooks-apps#1]
Looks like nice app. However, when I follow the README instructions to clone and run I receive the following error when doing the `panel serve` step:

ValueError: did not find a match in any of xarray's currently installed IO backends ['scipy']. Consider explicitly selecting one of the installed engines via the ``engine`` parameter, or installing additional IO dependencies, see:
https://docs.xarray.dev/en/stable/getting-started-guide/installing.html
https://docs.xarray.dev/en/stable/user-guide/io.html

I created a new conda environment before running the `pip install` step. I tried with both Python 3.11 and 3.10. I have a Macbook Pro with a M1 processor.

**Comment by ivandorte** [comment]
@hsparra Based on your error message, it looks like you are missing the scipy dependency. You can install it by running: `conda install scipy`.
Let me know if this solution works.

**Comment by hsparra** [comment]
@ivandorte, the issue ended up being I did not follow your instructions exactly. I used the `git clone https...` method since the instruction gave a security message of unable to verify signature when using the `git clone git@github.com...` method. Apparently the http approach converted the file to ascii text encoding. When I downloaded the datafile by itself and used it everything ran!

Thank you for sharing this great example!
